<!--
This initialize the camera
-->

<launch>
  <include file="$(find realsense2_camera)/launch/rs_aligned_depth.launch" />

  <node pkg="yolact_ros" type="yolact_ros" name="yolact_ros" output="screen">
    <param name="model_path" value="$(find yolact_ros)/scripts/yolact/weights/yolact_base_54_800000.pth" />
    <param name="image_topic" value="/camera/color/image_raw" />
    <param name="use_compressed_image" value="True" />
    <param name="publish_visualization" value="True" />
    <param name="publish_detecitons" value="True" />
    <param name="display_visualization" value="True" />
    <param name="display_fps" value="True" />
  </node>


  <!--
    <node pkg="prosthetic_hand" type="camera_node" name="camera_node" output="screen" />
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find prosthetic_hand)/rviz/config_file.rviz" />
  -->
</launch>
